### ISSUE TITLE
MariaDB Option

### ISSUE THREAD

**Issue by inetbiz** [serghey-rodin/vesta#1204]
Please add mariadb stable as an option to install for databases?

**Comment by Skamasle** [comment]
Will be nice, but this change is very easy after vesta was installed

Always you can run centos 7, this comes with mariadb by default.

**Comment by kevit-development** [comment]
> Always you can run centos 7, this comes with mariadb by default.

CentOS 7 comes with PHP 5.6, while Ubuntu comes with PHP7.

I'd rather have PHP7 with MySQL than PHP5.6 with MariaDB.

That being said, I would love the option to select either MariaDB or MySQL as an install option since MariaDB is faster for bigger queries/databases.

**Comment by huloza** [comment]
You can install mariadb modifying the installer before executing, remove the mysql lines and add the mariadb ones.

Vesta installs the OS stack.

**Comment by plutocrat** [comment]
Its actually quite a hard job to replace mysql with mariadb after initial installation, on Ubuntu 16 at least. This is because mysql version is 5.7 whereas mariadb version is 10.1, which is based on mysql 5.6. There were many big changes between 5.6 and 5.7, including the renaming of the password field! 
So yes, as someone who spent a long time converting between the two, I'd like to see an option in the installer script too. 
If anyone is considering doing this, install Maria db as early as possible, either by editing the script, or by installing manually and then removing mysql install from the script.

**Comment by anton-reutov** [comment]
@inetbiz for all OS by default ?

**Comment by kevzdev** [comment]
Yes @anton-reutov https://github.com/serghey-rodin/vesta/issues/1204#issuecomment-328803571

**Comment by dalu** [comment]
not default pls. this would be a nightmare.

have the option to chose which one would like to install.
mysql is the better piece of software mariadb is a political pick
if you want to chose a winner between the two go for percona server and make that default.

**Comment by inetbiz** [comment]
@anton-reutov not by default. Just as option.

**Comment by tgd1973** [comment]
How can I install in Ubuntu 16.04 the MariaDB with  installer  `vst-install-ubuntu.sh` instead of MySQL? I thank those who answer me.

**Comment by malsony** [comment]
I suggest provide an option to administrators to choose MySQL or MariaDB, no matter what OS they are using.
